docs/configuration/extract-codacy-logs.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+#### Helper functions ####
+
+usage()
+{
+  echo "Usage: $0 -n NAMESPACE "
+  exit 2
+}
+
+set_variable()
+{
+  local varname=$1
+  shift
+  if [ -z "${!varname}" ]; then
+    eval "$varname=\"$@\""
+  else
+    echo "Error: $varname already set"
+    usage
+  fi
+}
+
+
+#### Log extraction script ####
+
+while getopts "n:" option; do
+	case ${option} in
+		n )
```

Comment:
We should add an optional flag for only the last 3 days of logs, because in most cases we will not need the full 7 days.